tests/modules/object-cache/health-check/health-check-test.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Tests for webp-uploads module.
+ *
+ * @package performance-lab
+ * @group webp-uploads
```

Comment:
Not the right group :)

Let's use the module slug.

Reply:
Fixed in https://github.com/WordPress/performance/pull/111/commits/831c85ce6144314002636708c77c5b06554e55b6.